pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -490,9 +494,12 @@ func (bfs *Service) exiter() {
 	}
 	// Closing signal may come from anyone, but only once.
 	force := <-bfs.quit
+	modeStr := "Blocks"
+	if bfs.operationMode == Headers {
+		modeStr = "Headers"
+	}
 	bfs.log.Info("shutting down NeoFS BlockFetcher service",
-		zap.Bool("force", force),
-	)
+		zap.Bool("force", force), zap.String("mode", modeStr))
```

Comment:
Remove `	s.log.Info("NeoFS HeaderFetcher finished state sync headers downloading")` and `s.log.Info("NeoFS BlockFetcher finished state sync block downloading")`.